### PR TITLE
Add prison preset

### LINKED ISSUE #3070: Add a Preset for Prisons
**Preset Request**. [Prisons](http://wiki.openstreetmap.org/wiki/Tag:amenity%3Dprison) are currently shown on the map, [in a stylish gray zebra pattern](https://github.com/gravitystorm/openstreetmap-carto/pull/772), yet there is no easy way to tag them in iD.

The related tags for prison are `name`, `operator`, and `access`. Users often trace the barrier fence as the prison boundary, so the `barrier=fence` tag is useful in combination.

2016-04-17 17_56_58-openstreetmap

### PR DESCRIPTION
**Pull request by kepta**
(closes #3070)

### CODE REVIEW

**Comment by pnorman** [comment]
You should avoid having a linear tag (barrier=fence) on the same object as an area (amenity=prison), and this preset would encourage that.

File: data/presets/presets/amenity/prison.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        "area"
+    ],
+    "terms": [
+        "prison",
+        "cell",
+        "jail"
+    ],
+    "tags": {
+        "amenity": "prison",
+        "barrier": "fence"
```
Comment (bhousel) [inline data/presets/presets/amenity/prison.json]: Yeah what @pnorman said - remove `barrier: fence` from here.
People should tag the fence as a separate element, or of they really want to they can add this tag to the amenity using the raw tag editor, but we can't automatically assume that prisons are also fences.

File: data/presets/presets/amenity/prison.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "fields": [
+        "operator",
+        "access"
+    ],
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "terms": [
+        "prison",
```
Comment (bhousel) [inline data/presets/presets/amenity/prison.json]: Don't need to add "prison" as a search term if it is the tag value - it's already included.  The other terms you added are 👍 ..

File: data/presets/presets.json [hunk 3]
```diff
@@ -1507,6 +1507,25 @@
         },
         "name": "Post Office"
     },
+    "amenity/prison": {
+        "icon": "prison",
+        "fields": [
+            "operator",
+            "access"
```
Comment (bhousel) [inline data/presets/presets.json]: I think this should be `address` not `access`?

Comment (bhousel) [inline data/presets/presets.json]: I made this change and merged..

**Pull request merged**